ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/local_users.schema.yml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  local_users:
+    type: list
+    primary_key: name
+    convert_types:
+      - dict
+    display_name: Local-Users
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          display_name: Local-user Name
+        privilege:
+          type: int
+          required: true
+          display_name: Privilege
```

Comment:
Display_name is only needed if the name is different than the output of `key.replace('_',' ').title()`.